src/Error/ErrorProvider.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Digia\GraphQL\Error;
+
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+class ErrorProvider extends AbstractServiceProvider
+{
+    /**
+     * @var array
+     */
+    protected $provides = [
+        ErrorHandlerInterface::class,
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        $this->container->add(ErrorHandlerInterface::class, ErrorHandler::class, true/* $shared */);
```

Comment:
Since you merged the league/container update PR this needs updating too I guess?

Reply:
True, I'll change this.